### ISSUE TITLE
Wrong results count when zooming on whole world

### ISSUE THREAD

**Issue by fbunoz** [c2corg/v6_api#538]
When we zoom out to show the whole world, and when we use the map extend to filter, the result count should be the same or a bit less than not using the map extend filter.
But it is not : the count is few hundred only.

Ex : all WP on the world : http://www.demov6.camptocamp.org/waypoints#bbox=-22837631%252C-44634816%252C29447742%252C31601641
527 results (on my screen) instead of more than 30000.

**Comment by fbunoz** [comment]
Sorry, I created the issue on API, but it maybe more relevent for UI, but maybe not :-) I let you decide.

**Comment by asaunier** [comment]
Interesting :P 

Perhaps it's related to the fact your map extent is too big an covers several times the world => incorrect map extent?

If you zoom a bit and only have the world once, it seems OK:
http://www.demov6.camptocamp.org/waypoints#bbox=-18337019%252C-27219404%252C16572078%252C29527446

**Comment by tsauerwein** [comment]
It's likely a reprojection issue between 3857 <-> 4326.

**Comment by fbunoz** [comment]
> Perhaps it's related to the fact your map extent is too big an covers several times the world => incorrect map extent?

Yes, but on a 1920px screen, I couldn't have the whole world without the bug.
The bug disappear only when the extend is 90-95% of the world, then the results are not all the georeferenced doc in the DB.
